d2.config.js

```diff
@@ -0,0 +1,11 @@
+const config = {
```

Comment:
Maybe we should add a description as well: https://platform.dhis2.nu/#/config/d2-config-js-reference (I'm not sure if that's a requirement).

Reply:
It's not required, but I added one anyway.